<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Object Detection</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Background/Background/logo.png</normaloff>:/Background/Background/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>921</width>
      <height>391</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgba( 14, 13, 13, 0.55 );
box-shadow: 0 8px 32px 0 rgba( 31, 38, 135, 0.37 );
backdrop-filter: blur( 20px );
-webkit-backdrop-filter: blur( 20px );
border: 1px solid rgba( 255, 255, 255, 0.18 );
border: opacity(0%);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>581</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei Light</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgba( 76, 76, 76, 0.55 );
color: white;
box-shadow: 0 8px 32px 0 rgba( 31, 38, 135, 0.37 );
backdrop-filter: blur( 20px );
-webkit-backdrop-filter: blur( 20px );
border: opacity(0%);
</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Enter RTSP URL</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>50</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>13</pointsize>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgba( 76, 76, 76, 0.55 );
color: white;
box-shadow: 0 8px 32px 0 rgba( 31, 38, 135, 0.37 );
backdrop-filter: blur( 20px );
-webkit-backdrop-filter: blur( 20px );
border: 1px solid rgba( 255, 255, 255, 0.18 );
border: opacity(0%);</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
    <property name="autoRepeatDelay">
     <number>100</number>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>-260</x>
      <y>-20</y>
      <width>1341</width>
      <height>781</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Background/Background/Background.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="screenshot">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgba( 76, 76, 76, 0.55 );
box-shadow: 0 8px 32px 0 rgba( 31, 38, 135, 0.37 );
backdrop-filter: blur( 20px );
-webkit-backdrop-filter: blur( 20px );
border: 1px solid rgba( 255, 255, 255, 0.18 );
border: opacity(0%);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Icons/Icons/landscapev1.png</normaloff>:/Icons/Icons/landscapev1.png</iconset>
    </property>
    <property name="shortcut">
     <string>S</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="close_button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>0</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Icons/Icons/cancel (1).png</normaloff>:/Icons/Icons/cancel (1).png</iconset>
    </property>
    <property name="shortcut">
     <string>P</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
    <property name="autoRepeatInterval">
     <number>300</number>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="minimize_button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>0</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Icons/Icons/minimize-sign.png</normaloff>:/Icons/Icons/minimize-sign.png</iconset>
    </property>
    <property name="shortcut">
     <string>P</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
    <property name="autoRepeatInterval">
     <number>300</number>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Background/Background/logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="filter_truck">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>500</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Exan</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {color: white;}</string>
    </property>
    <property name="text">
     <string>Truck</string>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_filter">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>500</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Exan</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: white;}</string>
    </property>
    <property name="text">
     <string>Object Detection Filter |</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="filter_car">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>500</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Exan</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {color: white;}</string>
    </property>
    <property name="text">
     <string>Car</string>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="filter_motorbike">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>500</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Exan</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {color: white;}</string>
    </property>
    <property name="text">
     <string>Motorbike</string>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="filter_boat">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>500</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Exan</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {color: white;}</string>
    </property>
    <property name="text">
     <string>Boat</string>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="filter_person">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>500</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Exan</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {color: white;}</string>
    </property>
    <property name="text">
     <string>Person</string>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="selectFileButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>560</y>
      <width>125</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgba( 76, 76, 76, 0.55 );
color: white;
box-shadow: 0 8px 32px 0 rgba( 31, 38, 135, 0.37 );
backdrop-filter: blur( 20px );
-webkit-backdrop-filter: blur( 20px );
border: 1px solid rgba( 255, 255, 255, 0.18 );
border: opacity(0%);</string>
    </property>
    <property name="text">
     <string>Set File Path</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="uploadUrlLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>600</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgba( 76, 76, 76, 0.55 );
color: white;
box-shadow: 0 8px 32px 0 rgba( 31, 38, 135, 0.37 );
backdrop-filter: blur( 20px );
-webkit-backdrop-filter: blur( 20px );
border: 1px solid rgba( 255, 255, 255, 0.18 );
border: opacity(0%);</string>
    </property>
    <property name="placeholderText">
     <string> Enter upload url</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileNameLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>560</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgba( 76, 76, 76, 0.55 );
color: white;
box-shadow: 0 8px 32px 0 rgba( 31, 38, 135, 0.37 );
backdrop-filter: blur( 20px );
-webkit-backdrop-filter: blur( 20px );
border: 1px solid rgba( 255, 255, 255, 0.18 );
border: opacity(0%);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>File Path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uploadButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>630</y>
      <width>125</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(10, 0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>630</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Centauri</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: dark grey;
color: white;
border: opacity(0%);
</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Exan</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgba( 76, 76, 76, 0.55 );
color: white;
box-shadow: 0 8px 32px 0 rgba( 31, 38, 135, 0.37 );
backdrop-filter: blur( 20px );
-webkit-backdrop-filter: blur( 20px );
border: 1px solid rgba( 255, 255, 255, 0.18 );
border: opacity(0%);</string>
    </property>
    <item>
     <property name="text">
      <string>CPU</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GPU</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>600</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
border: opacity(0%);
</string>
    </property>
    <property name="text">
     <string>Enter FTP Url :</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="screenShot_checkBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>550</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
</string>
    </property>
    <property name="text">
     <string>Take Screenshot</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="source_comboBox">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Exan</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgba( 76, 76, 76, 0.55 );
color: white;
box-shadow: 0 8px 32px 0 rgba( 31, 38, 135, 0.37 );
backdrop-filter: blur( 20px );
-webkit-backdrop-filter: blur( 20px );
border: 1px solid rgba( 255, 255, 255, 0.18 );
border: opacity(0%);</string>
    </property>
    <item>
     <property name="text">
      <string>RTSP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Video</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Image</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Camera</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>11</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgba( 76, 76, 76, 0.55 );
color: white;
box-shadow: 0 8px 32px 0 rgba( 31, 38, 135, 0.37 );
backdrop-filter: blur( 20px );
-webkit-backdrop-filter: blur( 20px );
border: 1px solid rgba( 255, 255, 255, 0.18 );
border: opacity(0%);</string>
    </property>
    <property name="text">
     <string>Choose File</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="TCP_checkBox">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>570</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
border: opacity(0%);</string>
    </property>
    <property name="text">
     <string>Alert to TCP</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="FTP_checkBox">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>590</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
border: opacity(0%);
</string>
    </property>
    <property name="text">
     <string>Alert to FTP</string>
    </property>
   </widget>
   <zorder>progressBar</zorder>
   <zorder>background</zorder>
   <zorder>lineEdit</zorder>
   <zorder>start_button</zorder>
   <zorder>screenshot</zorder>
   <zorder>close_button</zorder>
   <zorder>minimize_button</zorder>
   <zorder>filter_truck</zorder>
   <zorder>label_filter</zorder>
   <zorder>filter_car</zorder>
   <zorder>filter_motorbike</zorder>
   <zorder>filter_boat</zorder>
   <zorder>filter_person</zorder>
   <zorder>selectFileButton</zorder>
   <zorder>uploadUrlLineEdit</zorder>
   <zorder>fileNameLineEdit</zorder>
   <zorder>uploadButton</zorder>
   <zorder>comboBox</zorder>
   <zorder>label_2</zorder>
   <zorder>screenShot_checkBox</zorder>
   <zorder>source_comboBox</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>TCP_checkBox</zorder>
   <zorder>FTP_checkBox</zorder>
   <zorder>label</zorder>
   <zorder>graphicsView</zorder>
  </widget>
  <action name="actionPlay">
   <property name="icon">
    <iconset>
     <normaloff>:/Images/Icons/3d-cube.png</normaloff>:/Images/Icons/3d-cube.png</iconset>
   </property>
   <property name="text">
    <string>Player</string>
   </property>
   <property name="iconText">
    <string>Player</string>
   </property>
   <property name="toolTip">
    <string>Player</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>close_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>805</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>806</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minimize_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>758</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>629</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>start_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
